doc/ch-grow_desc.rst

```diff
@@ -13,436 +13,5 @@ Synopsis
-      directories.
-
-   2. If within a directory named in sources, symlinks are copied as symlinks.
-
-We expect the following differences to be permanent:
-
-* Wildcards use Python glob semantics, not the Go semantics.
-
-* :code:`COPY --chown` is ignored, because it doesn't make sense in an
-  unprivileged build.
-
-Features we do not plan to support
-----------------------------------
-
-* Parser directives are not supported. We have not identified a need for any
-  of them.
-
-* :code:`EXPOSE`: Charliecloud does not use the network namespace, so
-  containerized processes can simply listen on a host port like other
-  unprivileged processes.
-
-* :code:`HEALTHCHECK`: This instruction's main use case is monitoring server
-  processes rather than applications. Also, implementing it requires a
-  container supervisor daemon, which we have no plans to add.
-
-* :code:`MAINTAINER` is deprecated.
-
-* :code:`STOPSIGNAL` requires a container supervisor daemon process, which we
-  have no plans to add.
-
-* :code:`USER` does not make sense for unprivileged builds.
-
-* :code:`VOLUME`: This instruction is not currently supported. Charliecloud
-  has good support for bind mounts; we anticipate that it will continue to
-  focus on that and will not introduce the volume management features that
-  Docker has.
-
-
-Environment variables
-=====================
-
-.. include:: py_env.rst
-
-
-Examples
-========
-
-:code:`build`
--------------
-
-Build image :code:`bar` using :code:`./foo/bar/Dockerfile` and context
-directory :code:`./foo/bar`::
-
-   $ ch-grow build -t bar -f ./foo/bar/Dockerfile ./foo/bar
-   [...]
-   grown in 4 instructions: bar
-
-Same, but infer the image name and Dockerfile from the context directory
-path::
-
-   $ ch-grow build ./foo/bar
-   [...]
-   grown in 4 instructions: bar
-
-Build using humongous vendor compilers you want to bind-mount instead of
-installing into a layer::
-
-   $ ch-grow build --bind /opt/bigvendor:/opt .
-   $ cat Dockerfile
-   FROM centos:7
-
-   RUN /opt/bin/cc hello.c
-   #COPY /opt/lib/*.so /usr/local/lib   # fail: COPY doesn't bind mount
-   RUN cp /opt/lib/*.so /usr/local/lib  # possible workaround
-   RUN ldconfig
-
-:code:`pull`
-------------
-
-Download the Debian Buster image and place it in the storage directory::
-
-  $ ch-grow pull debian:buster
-  pulling image:   debian:buster
-
-  manifest: downloading
-  layer 1/1: d6ff36c: downloading
-  layer 1/1: d6ff36c: listing
-  validating tarball members
-  resolving whiteouts
-  flattening image
-  layer 1/1: d6ff36c: extracting
-  done
-
-Same, except place the image in :code:`/tmp/buster`::
-
-   $ ch-grow pull debian:buster /tmp/buster
-   [...]
-   $ ls /tmp/buster
-   bin   dev  home  lib64  mnt  proc  run   srv  tmp  var
-   boot  etc  lib   media  opt  root  sbin  sys  usr
-
-..  LocalWords:  tmpfs'es bigvendor
+:code:`ch-grow` is deprecated and will be removed in a future release. See
+replacement :code:`ch-image`.
```

Comment:
I think this should be framed as a renaming; as written, it sounds like a new tool with a new name. Then say the old name is being kept as an alias for now but will be removed in 0.23.